src/coffee/repeater-task-queue.coffee

```diff
@@ -0,0 +1,73 @@
+{ Repeater } = require 'sphere-node-utils'
+Promise = require 'bluebird'
+util = require 'util'
+TaskQueue = require './task-queue'
+
+# Response messages which will be handled by `RepeaterTaskQueue`
+retryKeywords = [
```

Comment:
@emmenko Not all of the errors in the list return a proper error code. E.g. in some cases we got
```
"code": "EAI_AGAIN",
"errno": "EAI_AGAIN",
}
```
and ENOTFOUND doesn't even contain these properties.